image_LICENSE.txt

```diff
@@ -22,46 +22,22 @@ docs/source/tutorials/images:
               mpl_figure_editor.png | Enthought
                   traits_thread.png | Enthought
 
+integrationtests/ui/images:
```

Comment:
`integrationtests` is not distributed, right?

Reply:
Hmmm, it is, which is surprising to me. That's orthogonal then.

Reply:
yeah but i'm not sure if we should have attribution for *all* images in the github repository or attribution only for the images which are distributedvia traitsui.

Reply:
either way, we can actually remove them because at the top of the license file, we see the following

> "Unless stated in this file, icons are the work of Enthought, and are released under a 3 clause BSD license."

I went ahead and remove the section. @kitchoi do you think I should add it back?